### PR TITLE
project pet class

### PR DESCRIPTION
**Pull request by rukshand2001**

### CODE REVIEW

File: src/main/java/com/rukshan/petcare/Pet/PetController.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.util.List;
+
+@RestController
+public class PetController {
+
+    @Autowired
+    private PetServices petservices;
+
+    @GetMapping("/Dummy")
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetController.java]:
```suggestion
    @GetMapping("/ping")
```

File: src/main/java/com/rukshan/petcare/Pet/PetController.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public List<Pet>getAllPet(){
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetController.java]:
```suggestion
    public Message ping(){
```

File: src/main/java/com/rukshan/petcare/Pet/PetController.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        return petservices.getAllPet();
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetController.java]:
```suggestion
        return petservices.ping();
```

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package com.rukshan.petcare.Pet;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java]: Add extra space after this

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+@Service
+public class PetServices {
+    //String dummy = "hello pet";
+    private List<Pet>Dummy = Arrays.asList(
+            new Pet("Hello pet")
+    );
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java]: Why ArrayList ?

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+    public  List<Pet> getAllPet(){
+
+        return Dummy;
+    }
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java]: Just return a Message object

File: src/main/java/com/rukshan/petcare/Pet/Pet.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    public String getDummy() {
+        return Dummy;
+    }
+
+    public void setDummy(String dummy) {
+        Dummy = dummy;
+    }
+}
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/Pet.java]:
```suggestion
public class Message {

    private String message;
    private int status;

    public Message(String message, int status) {
       this.message = message;
       this.status = status;
    }

    public void setMessage(String message){
        this.message=message;
    }

    public String getMessage() {
        return this.message;
    }

    public void setStatus(int status) {
        this.status = status;
    }
    
     public int getStatus() {
        return this.status;
    }
}
```

Comment (aselarbd) [reply src/main/java/com/rukshan/petcare/Pet/Pet.java]: Move message class to a common location

Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/Pet.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java (hunk 5 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetServices.java]: removed it

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java (hunk 4 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetServices.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetController.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetController.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetController.java (hunk 3 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetController.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetController.java (hunk 1 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetController.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java (hunk 6 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetServices.java]: done

File: src/main/java/com/rukshan/petcare/Pet/Pet.java [hunk 8]
```diff
@@ -0,0 +1,6 @@
+package com.rukshan.petcare.Pet;
+
+public class Pet {
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/Pet.java]: remove this file

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class PetServices {
+
+    public  Message ping(){
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java:9]: Add a comment here explaining what are you doing.

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+        return ping();
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java]: Did you test this atleast ? why calling ping() hear

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java [hunk 11]
```diff
@@ -0,0 +1,17 @@
+package com.rukshan.petcare.Pet;
+
+import com.rukshan.petcare.utils.Message;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/Pet/PetServices.java]: Why aren't you remove these imports?

File: src/main/java/com/rukshan/petcare/Pet/Pet.java (hunk 8 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/Pet.java]: done

File: src/main/java/com/rukshan/petcare/Pet/PetServices.java (hunk 11 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/Pet/PetServices.java]: done

**Review by aselarbd — APPROVED** [review]

**Pull request merged**